packages/vite/src/types.ts

```diff
@@ -28,4 +28,10 @@ export interface VitePluginConfig<Theme extends {} = {}> extends UserConfig<Them
    * @default false
    */
   transformCSS?: boolean | 'pre' | 'post'
+  /**
+   *
+   * make the generate css processed by postcss(https://vitejs.dev/guide/features.html#postcss)
+   * @default false
```

Comment:
```suggestion
   * @default true
```

I guess we could enable this by default to align the dev and build